.github/workflows/cts_ci.yml

```diff
@@ -136,6 +136,7 @@ jobs:
           submodules: true
       - name: Configure CMake
         working-directory: ${{ env.container-workspace }}
+        shell: bash
         run: |
           bash /scripts/configure.sh \
```

Comment:
Technically, this is not needed too. `run` command invokes configuration script from bash.
As an alternative, we can keep shell configuration in GitHub Actions script and simplify the `run` command by dropping `bash` command.

Reply:
Yeah, another leftover from from the patching, which used bash `if [[]]` syntax.